pom.xml

```
@@ -56,8 +56,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.8.0</version>
                     <configuration>
-                        <source>${jdk.version}</source>
-                        <target>${jdk.version}</target>
+                        <source>1.8</source>
+                        <target>1.8</target>
```

return the original version

---

how to find out what was the version before I changed to 1.8?

---

Якщо у тебе є якісь питання по задачам їх потрібно задавати в slack у чатику jv_chat або в цьому чатику з командою. Комунікаця на github дуже не зручна та розтягнута у часі. 
```suggestion
                        <source>${jdk.version}</source>
                        <target>${jdk.version}</target>
```